Makefile

```diff
@@ -384,7 +384,7 @@ launch-logging: check-kubectl check-kubectl-connection
 	kubectl --namespace=monitoring port-forward `kubectl --namespace=monitoring get pods -l k8s-app=kibana-logging -o json | jq '.items[0].metadata.name' -r` 35601:5601 &
 
 launch-loki:
-	helm repo add loki https://grafana.github.io/loki/charts
+	helm repo add --force-update loki https://grafana.github.io/loki/charts
```

Comment:
I don't totally understand the implications of `--force-update` for Helm. But this seem to work in CI and I assume this just gets us around some potential errors so seems reasonable to me.

Reply:
Just noticed this :grimacing: . It gets us around a bug in Travis's retry logic. Basically, if the tests fail, Travis retries the whole test script now (a change Luke made to help with our flaky tests), but that means running this command twice too. The first time it succeeds, but during retries it fails with `repo "loki" already exists` or something like that. The `--force-update` command just prevents helm from giving us that error and forces it to recreate the repo anyway